generators/_node/index.js

```diff
@@ -173,6 +173,16 @@ class PackageJSONGenerator extends Base {
       cwd: this.destinationPath(this.cwd)
     });
 
+    this.spawnCommandSync('git', ['add', '--all'], {
+      cwd: this.destinationPath(this.cwd)
+    });
+    this.spawnCommandSync('git', ['commit', '-am', '"Initial commit"'], {
```

Comment:
I think the correct statement would be 'git commit --allow-empty -m 'Initial commit'